python/cac_tripplanner/templates/place-detail.html

```diff
@@ -1,6 +1,30 @@
 
 {% extends "base.html" %}
 {% load staticfiles %}
+
+{% block pagetitle %}
+<title>{% block title %}GoPhillyGo | {{ destination.name }}{% endblock %}</title>
+
+<meta property="og:title" content="{{ destination.name }}" />
+<meta name="twitter:title" content="{{ destination.name }}" />
+{% endblock %}
+
+{% block extrametatags %}
+<meta property="fb:app_id" content="{{ fb_app_id }}" />
+<meta property="og:url"
+    content="{% if request.is_secure %}https{% else %}http{% endif %}://{{ request.META.HTTP_HOST }}" />
```

Comment:
Is there no `request.protocol`? Might be cleaner, but then you have the potential for `file://` to sneak in there. :man_shrugging: